crt_portal/static/sass/custom/modal.scss

```diff
@@ -30,12 +30,30 @@ body.is-modal {
   .modal-content {
     position: relative;
     background: white;
-    width: 80%;
-    height: 60%;
-    padding: 2rem;
     @include u-radius('md');
     @include u-shadow(3);
+  }
+
+  .modal-content--small {
+    @include at-media-max(mobile-lg) {
```

Comment:
You could remove this media query wrapper completely so that the default sizing for the modal is the mobile sizing.

Reply:
Thanks. Yea, and I think I messed this part up -- one of these queries should have been `tablet`. Fixed. (Default is mobile sizing)